examples/pipelines/fl_cross_silo_native/submit.py

```diff
@@ -23,6 +23,8 @@
 # to handle yaml config easily
 from omegaconf import OmegaConf
 
+# subgraph
+os.environ["AZURE_ML_CLI_PRIVATE_FEATURES_ENABLED"] = "true"
```

Comment:
If we have to do this, does this mean it's OK to put in a public repo?

Reply:
That's a good question. Let me cc @jfomhover (he might have an opinion on the same)